scripts/kind-e2e/ipam/deploy-ipam-2.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: submariner-ipam
+  namespace: submariner
+spec:
+  selector:
+    matchLabels:
+      app: submariner-ipam
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: submariner-ipam
+    spec:
+      containers:
+        - name: submariner-ipam
+          image: submariner-ipam:local
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: SUBMARINER_GLOBALCIDR
+              value: 169.254.1.0/24
```

Comment:
"/24" is too small a value. Lets use something like "/16" which is more realistic. I agree that this is only a test-case where we are not spawning even 255 pods/services, but using 16 allows us to be close to typical deployments and helps us in finding out any issues sooner.